docs/2_1_patching.md

```diff
@@ -8,16 +8,26 @@ The following pages will guide you through using ReVanced Manager to patch apps.
 2. Tap the + button in the bottom right corner
 3. Choose an app to patch[^1]
 4. Tap on the version of the app you want to patch[^2]
-5. Select the patches you want to apply
-6. Tap the Patch button
-7. Tap on the **Install** button
-   > **Note**: If you are rooted, you can mount the patched app on top of the original app.[^3]
-   > Optionally, you may export the patched app to storage using the options in the top right corner.
+5. Tap the 🪄 **Patch** button
+6. Tap on the 📲 **Install** button
+   > [!Note]
+   > If the **Install** Button isn't there, it could be that something went wrong. Refer to [3. ❔ Troubleshooting](3_troubleshooting.md)
+   > [!Tip]
+   > If you are rooted, you can mount the patched app on top of the original app.[^3]
+   > Optionally, you may export the patched app to storage using the options in the bottom left corner.
 
-[^1]: Non-root users may be prompted to select an APK from storage, in which case you have to source the APK file yourself. ReVanced does not provide any APK files.
+[^1]: You need to select an APK from storage, in which case you have to source the APK file yourself. ReVanced does not provide any APK files.
 [^2]: It is suggested to use the version with the most patches to get the most out of ReVanced.
 [^3]: Mounting the patched app on top of the original app will only work if the installed app version matches the version of the app selected in step 4. above.
 
+### Change patch selection and options
```

Comment:
Once again, this should not be a separate section, follow how the main branch  does it.